zdict/dictionaries/yandex.py

```diff
@@ -0,0 +1,66 @@
+import json
+
+from zdict.dictionary import DictBase
+from zdict.exceptions import NotFoundError
+from zdict.models import Record
+
+
+# The daily request limit is 1,000,000 characters
+# The monthly limit is 10,000,000 characters
+API_KEY = 'trnsl.1.1.20170826T075621Z.' \
```

Comment:
:/ I'm not a fan of escaping newline.
But I'm ok, if you do not like string concatenation.